test/integration/install_simple/controls/forseti.rb

```diff
@@ -178,6 +199,39 @@
   end
 end
 
+control 'forseti-no-public-ips' do
```

Comment:
(nit) Can you please put this control in a separate file? In an upcoming PR, I was going to move the forseti controls into a shared (symlink) file that can be leveraged by all tests. Since the private IP tests is something that will only be used by the install_simple tests, this would fail.